js/scenes/Settings.js

```diff
@@ -31,7 +31,12 @@ export default class Settings extends Component {
     });
   }
 
+  onNavigate(id) {
```

Comment:
NIT: Naming: This isn't a response/event to navigation but an instruction to navigate.

`onNavigate` -> `navigate`